### DISCUSSION TITLE
Have an option to hide the widget

### DISCUSSION THREAD

**Discussion by alexkubica** [dailydotdev/daily#941] [Feature Requests]
The widget can sometimes hide content I want to view on the website so it'll be nice to have an option to hide it.
For now the only workaround seems to be to allow the extension to only access daily.dev but that means I won't have the widget at all.

**Comment by idoshamun** [comment]
Hi @alexkubica,

Thanks for the suggestion. This can be very annoying I agree. We'll check what can be done.
